resources/todomvc/big-dom-generator/src/components/tree-area.jsx

```diff
@@ -8,10 +9,10 @@ const TreeItem = (props) => {
     const { treeNode, currentDepth } = props;
 
     const isExpandableItem = treeNode.type === "expandableItem";
-    const treeViewItemIsOpen = isExpandableItem && currentDepth < MAX_VISIBLE_TREE_VIEW_ITEM_DEPTH ? "is-open" : "";
+    const treeViewItemIsOpen = isExpandableItem && (currentDepth !== MAX_VISIBLE_TREE_VIEW_ITEM_DEPTH || treeNode.isDisplayNone);
```

Comment:
I don't understand this condition, it would be good to explain the logic of deciding if a node uses is-open or display-none or both.

Reply:
Added this comment "Every expandable TreeItem is open by default unless it is at the MAX_VISIBLE_TREE_VIEW_ITEM_DEPTH threshold. If it is marked as display none, set it as open so the `visibility: hidden` spectrum rule is removed.".